husarion_ugv_navigation/src/map_autosaver_node.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright 2024 Husarion sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "husarion_ugv_navigation/map_autosaver_node.hpp"
+
+namespace husarion_ugv_navigation {
+
+constexpr auto SAVE_MAP_CONNECTION_TIMEOUT = 2s;
+constexpr auto MIN_SAVE_MAP_PERIOD = 5s;
```

Comment:
I suggest to not use using namespace for in range of the class.
```suggestion
{
  using namespace std::chrono_literals;
  constexpr auto SAVE_MAP_CONNECTION_TIMEOUT = 2s;
  constexpr auto MIN_SAVE_MAP_PERIOD = 5s;
}
```

```suggestion
{
  constexpr auto SAVE_MAP_CONNECTION_TIMEOUT = std::chrono_literals::2s;
  constexpr auto MIN_SAVE_MAP_PERIOD =std::chrono_literals::5s;
}
```